src/v2/Apps/Fair/Routes/FairArtworks.tsx

```diff
@@ -0,0 +1,33 @@
+import { Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FairArtworks_fair } from "v2/__generated__/FairArtworks_fair.graphql"
+import { FairAboveFoldFragmentContainer as FairAboveFold } from "../Components/FairAboveFold"
+
+interface FairArtworksProps {
+  fair: FairArtworks_fair
+}
+
+const FairArtworks: React.FC<FairArtworksProps> = ({ fair }) => {
+  return (
+    <>
+      <FairAboveFold fair={fair} />
```

Comment:
I'm surprised to see this component (`FairAboveFold`) present in each of these 'route-leve' HOC's. Should the `FairAboveFold` component move up to `FairApp`?

Reply:
YES

Reply:
Wait, actually — no: because of the `/info` route

Reply:
Ah, yea, I was thinking that page wouldn't be a 'child' of the `FairApp`, but a sibling (and thus a wholly separate `FairInfoApp` (although the `...App` suffix feels a tad off since it's a fairly static page, but whatever)). In https://github.com/artsy/force/blob/0a478d342899b67f30c979a4a97a52182df8e9f2/src/v2/Apps/Fair/routes.tsx#L10-L11

Reply:
Though an alternative would be a `FairShell` component that accepted children.

Reply:
@dzucconi - this is how we do this in the [artist app](https://github.com/artsy/force/blob/master/src/v2/Apps/Artist/routes.tsx#L156), which we then [check here](https://github.com/artsy/force/blob/master/src/v2/Apps/Artist/ArtistApp.tsx#L68).

Reply:
👍